Symfony/CS/Fixer/Contrib/PrintToEchoFixer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class PrintToEchoFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $printTokens = $tokens->findGivenKind(T_PRINT);
+
+        foreach ($printTokens as $printIndex => $printToken) {
+            $prevToken = $tokens[$tokens->getPrevMeaningfulToken($printIndex)];
+            if ($prevToken->equalsAny(array(
+                    [T_RETURN, 'return'],
+                    [T_PLUS_EQUAL, '+='],
+                    [T_IS_EQUAL, '=='],
+                    [T_IS_IDENTICAL, '==='],
+                    '(',
+                    '=',
+                    ':',
+                    '.',
```

Comment:
@gharlan How would you manage the whitelist?

I mean, for most of case, the `print` function should be called with... nothing behind.

Maybe `Token` class has something like `isOperator`, will check that.